### DISCUSSION TITLE
Implement "max pp" attribute in `BeatmapAttributeText`

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22096] [Ideas] [answered]
Someone should implement "max pp" attribute for `BeatmapAttributeText` in skin editor because its the most valuable information to display. Also this attribute should be affected by selected mods.

Will look like this?
![image](https://user-images.githubusercontent.com/86934170/211369408-4e734031-bc5a-4421-b502-9acfd57e0bf7.png)

**Comment by frenzibyte** [comment]
If this were to be implemented, I would probably call it "max pp" or otherwise.

**Reply by peppy** [reply]
For sure. Because "if FC" is completely misleading - it could be a full combo with all 50s.

**Comment by cdwcgt** [comment]
https://user-images.githubusercontent.com/34775378/211538591-73c2b3a4-31ed-49d7-b289-16ceb379cada.mp4

This should be fine, I will open a pr ~~when I'm done writing the tests~~ forgive me. I really don't know where to test

**Reply by Loreos7** [reply]
OMG you're awesome. Looks amazing. Waiting for actual merging <3
Thank you for your efforts on my issue.

**Comment by Loreos7** [comment, marked as answer]
Implemented in https://github.com/ppy/osu/pull/30322
